src/backend/protocols/go.twitter/account.go

```diff
@@ -245,7 +258,7 @@ func (worker *AccountWorker) PollDM() {
 			err = worker.broker.ProcessInDM(worker.userAccount.userID, worker.userAccount.remoteID, &event, true)
 			if err != nil {
 				// something went wrong, forget this DM
-				log.WithError(err).Warnf("[AccountWorker %s] ProcessInDM failed for event : %+v", worker.userAccount.remoteID.String(), event)
+				log.WithError(err).Warnf("[AccountHandler %s] ProcessInDM failed for event : %+v", worker.userAccount.remoteID.String(), event)
```

Comment:
i Think it's an error and not a warning

Reply:
quoting yourself : « If program flow can continue but a strange situation found, so it's a warning ». That's precisely the case here.

Reply:
the processing of DM can't continue and flow stop here (for this DM), but yes the application will continue to run ;)

Reply:
what I wanted to say is that the func is not returning : 
* func aborts => error
* func continues => warning
what do you think of that ?

Reply:
yes you're right